state/core_access.go

```diff
@@ -51,9 +57,10 @@ type CoreAccessor struct {
 	signer *apptypes.KeyringSigner
 	getter libhead.Head[*header.ExtendedHeader]
 
-	queryCli   banktypes.QueryClient
-	stakingCli stakingtypes.QueryClient
-	rpcCli     rpcclient.ABCIClient
+	queryCli    banktypes.QueryClient
+	stakingCli  stakingtypes.QueryClient
+	feeGrantCli feegrant.QueryClient
```

Comment:
nit: cli naming is a bit confusing, I thought of CLI while reading the code, but its a short for Client. If you want to shorten it, let's use clnt